source/blog/2015-06-18-ember-data-1-13-released.md

```diff
@@ -0,0 +1,774 @@
+---
+title: Ember Data v1.13 Released
+author: Igor Terzic and the Ember Data Team
+tags: Recent Posts, Releases
+---
+
+We are proud to announce the release of Ember Data 1.13. This
+represents the first stable release of Ember Data since its creation
+over 3 years ago. 
+
+As we explained in the [THE EMBER 2.X PROJECT](http://emberjs.com/blog/2015/06/16/ember-project-at-2-0.html)
+blog post, going forward Ember Data will be syncing up it's releases and version numbers with Ember.js releases. 
+
+**Ember Data 1.13 is the first release of Ember Data that syncs its
+  version with a version of Ember.js.** It will be followed by Ember Data 2.0, which will be released alongside Ember.js 2.0 **Ember Data 1.13 is fully backwards compatible with Ember Data beta.19, allowing for a smooth upgrade path**
+
+###Ember Data 1.13 overview
+
+Ember Data 1.13 is a massive release we are very proud of.
+The highlight of the Ember Data 1.13 release is a total overhaul of Ember Data's internal data format and Serializer to follow JSON API. 
+
+Two years ago Tom Dale and Yehuda Katz [published](http://emberjs.com/blog/2013/05/03/ember-data-progress-update.html) a vision for how Ember Data should look in the future and articulated the need for a single, ubuqituous JSON API standard.
+
+We are very happy to see the vision for a single, ubuqituous JSON API standard come to life in the form of the [JSON API 1.0 release](jsonapi.org).
+
+JSON API and Ember Data have been interwined since JSON API's inception
+@DanGebhart gives the origins of JSON API in his 1.0 announcment:
+> Yehuda Katz wrote the first draft of the JSON API specification in May 2013 after hammering out the details in a long discussion with Steve Klabnik at RailsConf. JSON API began as a codification of the shared expectations of a single server library for Rails, ActiveModel::Serializers, and a single JavaScript client library, Ember Data
+
+While Ember Data has supported JSON API since one of early versions of JSON API through a community [adapter](https://github.com/kurko/ember-json-api) started by 
+@daliwali and maintained by @kurko, now that JSON API has reached 1.0 it is time for Ember Data to uphold it's part of the bargain and make using JSON API a first class experience in Ember Data. 
+
+We have done exactly that, and Ember Data 1.13 adds support throughout the ED stack for JSON Api:
+
+- **Ember Data 1.13 ships with a fully supported JSONApiAdapter and Serializer.** In 2.0, these will become the default Adapter and Serializer
+- **`JSONSerializer` and `RestSerializer` have been refactored and streamlined to return JSON API payloads**
+- **`store.push` now accepts JSON API compliant payload**
+
+Switching to the JSON API format unlocks many new features which will be added in the 2.0 cycle, including better first class pagination, filtering and metadata support.
```

Comment:
The combination of "better" and "first class" reads awkwardly to me. I would just strike "better" and stick with first-class.